packages/nimble-components/src/table/tests/table-matrix.stories.ts

```diff
@@ -57,6 +58,12 @@ export const tableThemeMatrix: Story = createMatrixThemeStory(
     createMatrix(component)
 );
 
+tableThemeMatrix.play = (): void => {
```

Comment:
>Updated storybook story for the table to have a button that loads rows into the table
>
>    Ideally, I would have liked to have this done automatically, but I couldn't find a way to do this because the play function was only called when the story initially loads, not when args are changed.

An unfortunate side effect of this is that the Chromatic snapshot for the non-matrix story is now an empty table. At least we still have test coverage for row/cell appearance via the matrix story, but it would be nicer if we could have the table be populated with data initially.

Could you describe the behavior in more detail if we add a `play` function which calls `setData()` to the non-matrix story? Would it start out with data (including in the Chromatic snapshot) but then clear it whenever the user changes an argument? Or does Storybook do something on load that causes the data to never be populated?

Reply:
We could implement `play` to call `setData()` on the non-matrix story. When you change anything in the control panel of storybook (which cases the table to re-render), the table data would be cleared and `play` would not be called again.

Reply:
Sounds like Milan's suggestion below might remove the need for a button. 

If that doesn't work for some reason then I'm actually now leaning back in the direction you initially proposed of not having a play function at all. Chromatic test coverage of the non-matrix story isn't that important and I think the user experience is more consistent if they learn to click the button from the very beginning.